src/notation/view/widgets/editstyle.ui

```diff
@@ -14089,11 +14157,6 @@ By default, they will be placed such as that their right end are at the same lev
   <tabstop>resetFrameSystemDistance</tabstop>
   <tabstop>lastSystemFillThreshold</tabstop>
   <tabstop>resetLastSystemFillThreshold</tabstop>
-  <tabstop>genClef</tabstop>
-  <tabstop>genKeysig</tabstop>
-  <tabstop>genCourtesyClef</tabstop>
-  <tabstop>genCourtesyTimesig</tabstop>
-  <tabstop>genCourtesyKeysig</tabstop>
```

Comment:
Make sure any new buttons have a sensible tab order. This can be done as follows:
1. Click this tiny button to enter tab order edit mode
    <img width="745" alt="Scherm­afbeelding 2024-01-12 om 00 14 41" src="https://github.com/musescore/MuseScore/assets/48658420/2908ca0b-2444-489f-9a8f-af22f73c3729">
2. On the previous page, look what's the last tab stop number: it's 131
    <img width="946" alt="Scherm­afbeelding 2024-01-12 om 00 17 11" src="https://github.com/musescore/MuseScore/assets/48658420/ad4df605-266e-40cb-a63c-ae8fadd4b0e0">
3. Right-click and choose "Tab Order List"
4. Drag the new buttons from the bottom of the list to just after item 131

    https://github.com/musescore/MuseScore/assets/48658420/0281f06f-6489-405f-b84d-a842b0809e43

5. Right-click the first item on the page and choose "start from here"
6. Click the items in the correct order. (misclicked? Use "start from here" again)

    https://github.com/musescore/MuseScore/assets/48658420/5bf2286c-ba41-4ea9-9424-877344ff8ad6

(and yes, it can be hyper frustrating)